ml-proto/test/memory.wasm

```diff
@@ -26,6 +27,17 @@
   (module (memory 100 1000 (segment 0 "a") (segment 2 "b") (segment 1 "c")))
   "data segment not disjoint and ordered")
 
+;; Test alignment annotation rules
+(assertinvalid (module (func (loadu.2.i8 (const.i32 0)))) "alignment too big")
```

Comment:
The design document isn't expected to be a complete enumeration of all invalid conditions, that's what `spec` is for and so this test *is* correct, wrt the spec as updated by this PR.  If you would like to disagree and argue that we *should* allow too-big alignment in the spec, that's another matter entirely, we shouldn't expect the high-level design repo to have to reiterate every single minute detail; that's not its purpose.